airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/integrations/base/ssh/SshBastionContainer.kt

```diff
@@ -51,10 +51,10 @@ class SshBastionContainer : AutoCloseable {
     }
 
     @Throws(IOException::class, InterruptedException::class)
-    fun getTunnelMethod(tunnelMethod: SshTunnel.TunnelMethod?, innerAddress: Boolean): JsonNode? {
+    fun getTunnelMethod(tunnelMethod: SshTunnel.TunnelMethod, innerAddress: Boolean): JsonNode? {
```

Comment:
Why nullable JsonNode ?  does Jsons.jsonNode doesn't guarantee empty JsonNode instead of null ?

Reply:
you're right. This is actually reported as a compiler warning.  I only focused on the spotbugs in this PR